vue-components/stories/Link.stories.ts

```diff
@@ -2,60 +2,60 @@ import Link from '@/components/Link';
 import { Component } from 'vue';
 
 export default {
-    component: Link,
-    // the `/` prefix in the title is needed for "Message" to appear as a folded navigation item, and not a headline
-    title: '/Link',
+	component: Link,
+	// the `/` prefix in the title is needed for "Message" to appear as a folded navigation item, and not a headline
+	title: '/Link',
 };
 
 export function basic( args: { content: string; underlined: boolean } ): Component {
-    return {
-        data(): object {
-            return { args };
-        },
-        components: { Link },
-        props: Object.keys( args ),
-        template: `
+	return {
+		data(): object {
+			return { args };
+		},
+		components: { Link },
+		props: Object.keys( args ),
+		template: `
 			<div>
 			<p><Link
                 :underlined="underlined"
```

Comment:
Somehow the indenting with spaces here is not raising an eslint issue? 🙃

Reply:
Meh, template literals are special, I guess. I won't attempt to fix that in this PR.

Reply:
Fair enough. But let's keep it in mind in case we ever come across a rule that addresses it :)